### ISSUE TITLE
try to find outspend for txo if it is spent (romanz/electrs backend)

### ISSUE THREAD

**Issue by antonilol** [mempool/mempool#1195]
### Description

when an output is spent, try finding the spending TX by scriptPubKey (query electrs) and fill in the outspend giving the browser a clickable link under the red arrow
https://github.com/mempool/mempool/issues/1088#issuecomment-1006762756

**Comment by wiz** [comment]
I don't think that is going to scale well, we currently do this on the official site and it is a lot of queries
